src/containerapp/azext_containerapp/containerapp_env_decorator.py

```diff
@@ -69,3 +75,27 @@ def get_argument_enable_workload_profiles(self):
 
     def get_argument_enable_dedicated_gpu(self):
         return self.get_param("enable_dedicated_gpu")
+
+    def get_argument_logs_dynamic_json_columns(self):
+        return self.get_param("logs_dynamic_json_columns")
+
+
+class ContainerappEnvPreviewUpdateDecorator(ContainerAppEnvUpdateDecorator):
+    def set_up_app_log_configuration(self):
+        logs_destination = self.get_argument_logs_destination()
+
+        if logs_destination:
```

Comment:
When using `az containerapp env update  --logs-destination azure-monitor`, we should set `logAnalyticsConfiguration `to `None`, otherwise it will throw following error from RP:
`(InvalidRequestParameterWithDetails) LogAnalyticsConfiguration is invalid.  When using Azure Monitor Log Destination, Log Analytics can only be configured under Resource Diagnotic settings`

Reply:
I fix it in PR https://github.com/Azure/azure-cli-extensions/pull/6972